app/models/admin.py

```diff
@@ -0,0 +1,13 @@
+from app.models import *
+
+class Admins(Base, UserMixin):

```

Comment:
One model for all types of user will be a better idea (since the most of the fields are share between them).

Reply:
`is_admin` flag will work here well